### ISSUE TITLE
Current Opponent And Current BPM In Wrong Layer

### ISSUE THREAD

**Issue by MushyMushroom1245** [JordanSantiagoYT/FNF-JS-Engine#488]
### Describe your bug here. If you are modding, try testing your bug in a clean version of the engine instead. Also, if you're using an older version of JSE, please try the latest version/action build. Also, be sure to check the pinned 'JS Engine: Known Issues' issue, to check if your issue hasn't already been found!

the current opponent/bpm event is usually on the camhud layer but now its on the camgame layer for some reason
![Screenshot 2024-08-13 103657](https://github.com/user-attachments/assets/5cecdc16-0414-4282-88e9-30208c7dd9c1)

![Screenshot 2024-08-13 103719](https://github.com/user-attachments/assets/4007426b-9802-452d-8ee4-ca127eb2d5b7)

link if you want to look at the events https://drive.google.com/drive/folders/1fmRLZ6wDH6pixZQMnPC45wHsoAnh446F?usp=sharing

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

no

### If you use 1.12.0 or earlier, did you have Optimized Chart Loading turned on?

None

### Did you check for any similar issues to what you're reporting? Check, then come back here. If there is a similar issue, then do not report the issue, otherwise it will be marked as a duplicate.

Yes

**Comment by NAEL2XD** [comment]
use 
```setObjectOrder('yourobjectname', num)```

**Comment by NAEL2XD** [comment]
here's the fix

Add BPM:
```lua
-- Event notes hooks
function onEvent(name, value1, value2)
        if name == 'Add BPM' then
                thing = value1
                green = tonumber(value2)
                makeLuaText('thebpm', "Current BPM: " .. thing, 1275, 2, 200)
                setTextSize('thebpm', 25);
                addLuaText('thebpm');
                if green == 1 then
                        setProperty('thebpm.color', getColorFromHex('00FF00'));
                        doTweenColor("thebpm", "thebpm", "FFFFFF", 0.4, "sineInOut")
                end
        end
        setObjectCamera("thebpm", "camHUD")
end
```

Show Current Opponent
```lua
-- Event notes hooks
function onEvent(name, value1, value2)
        if name == 'Show Current Opponent' then
                green = tonumber(value2)
                makeLuaText('theoppo', "Current Opponent: " .. value1, 1275, 2, 175)
                setTextSize('theoppo', 25);
                addLuaText('theoppo');
                if green == 1 then
                        setProperty('theoppo.color', getColorFromHex('00FF00'));
                        doTweenColor("theoppo", "theoppo", "FFFFFF", 0.4, "sineInOut")
                end
        end
        setObjectCamera("theoppo", "camHUD")
end```

**Comment by moxie-coder** [comment]
what he said basically

**Comment by MushyMushroom1245** [comment]
Alright Thanks

**Comment by JordanSantiagoYT** [comment]
by the way ive already fixed this issue with 1.32.0
